sway/config/output.c

```diff
@@ -306,6 +311,13 @@ bool apply_output_config(struct output_config *oc, struct sway_output *output) {
 		sway_log(SWAY_DEBUG, "Set %s scale to %f", oc->name, oc->scale);
 		wlr_output_set_scale(wlr_output, oc->scale);
 	}
+
+	if (oc && (oc->subpixel != WL_OUTPUT_SUBPIXEL_UNKNOWN || config->reloading)) {
+		sway_log(SWAY_DEBUG, "Set %s subpixel to %s", oc->name,
+			sway_wl_output_subpixel_to_string(oc->subpixel));
+		wlr_output_set_subpixel(wlr_output, oc->subpixel);
```

Comment:
Don't you need to damage the whole output here?

Reply:
Originally I put the damage in `wlr_output_set_subpixel()`. Fixed.